thoth/storages/graph/postgres.py

```diff
@@ -6765,3 +6790,15 @@ def get_python_package_version_all(self, analysis_document_id: str) -> List[Dict
             )
 
             return [{"package_name": i[1], "package_version": i[2], "location": i[0]} for i in query.all()]
+
+    def get_python_package_version_import_packages_all(self, import_name: str) -> str:
```

Comment:
return type should be `List[str]`

Reply:
fixed in 3f36204cb14d6f6cd76d9bdb7357c33dcfa5ad51